cmd/manager/main.go

```diff
@@ -319,11 +319,13 @@ func (f finalizerFunc) Finalize(ctx context.Context, obj client.Object) (crfinal
 func authFilePathIfPresent(logger logr.Logger) string {
 	_, err := os.Stat(authFilePath)
 	if os.IsNotExist(err) {
+		logger.Info("auth file not found, skipping configuration of global auth file", "path", authFilePath)
 		return ""
 	}
 	if err != nil {
-		logger.Error(err, "could not stat auth file path", "path", authFilePath)
+		logger.Error(err, "unable to stat auth file path", "path", authFilePath)
```

Comment:
`unable to stat auth file path` is kind of meaning less in a way. How would the user know that we are talking about os.stat and what does that mean.